plugins/modules/aci_l3out_hsrp_interface_profile.py

```diff
@@ -0,0 +1,317 @@
+description:
+- Manages L3Out HSRP interface profile on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+  interface_profile:
+    description:
+    - Name of an existing interface profile.
+    type: str
+    aliases: [ name, interface_profile_name, logical_interface ]
+  hsrp_policy:
+    description:
+    - Name of an existing hsrp interface policy.
+    type: str
+    aliases: [ name, hsrp_policy_name ]
+  version:
+    description:
+    - The version of the compatibility catalog.
+    type: str
+    choices: [ v1, v2 ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+notes:
+- The C(tenant), C(l3out), C(logical_node_profile) and C(logical_interface_profile) must exist before using this module in your playbook.
+  The M(cisco.aci.aci_tenant), M(cisco.aci.aci_l3out), M(cisco.aci.aci_l3out_logical_node_profile) and M(cisco.aci.aci_l3out_logical_interface_profile)
+  can be used for this.
+seealso:
+- module: aci_tenant
+- module: aci_l3out
+- module: aci_l3out_logical_node_profile
+- module: aci_l3out_logical_interface_profile
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC classes
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Shreyas Srish (@shrsr)
+"""
+
+EXAMPLES = r"""
+- name: Add a new hsrp interface profile
+  cisco.aci.aci_l3out_hsrp_interface_profile:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: my_tenant
+    l3out: my_l3out
+    node_profile: my_node_profile
+    interface_profile: my_interface_profile
+    hsrp_policy: default
+    state: present
+  delegate_to: localhost
+
+- name: Delete a hsrp interface profile
+  cisco.aci.aci_l3out_hsrp_interface_profile:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: my_tenant
+    l3out: my_l3out
+    node_profile: my_node_profile
+    interface_profile: my_interface_profile
+    hsrp_policy: default
+    state: absent
+  delegate_to: localhost
+
+- name: Query a hsrp interface profile
+  cisco.aci.aci_l3out_hsrp_interface_profile:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: my_tenant
+    l3out: my_l3out
+    node_profile: my_node_profile
+    interface_profile: my_interface_profile
+    hsrp_policy: default
+    state: query
+  delegate_to: localhost
+  register: query_result
+"""
```

Comment:
aci_l3out_hsrp_interface_profile - supports query all objects?

Reply:
No. Only one can be created. It's a placeholder object.